bincapz.go

```diff
@@ -50,6 +51,8 @@ func main() {
 	statsFlag := flag.Bool("stats", false, "Show statistics about the scan")
 	thirdPartyFlag := flag.Bool("third-party", true, "Include third-party rules, which may have licensing restrictions")
 	verboseFlag := flag.Bool("verbose", false, "Emit verbose logging messages to stderr")
+  versionFlag := flag.Bool("version", false, "show version information")
```

Comment:
Capitalize "Show" for consistency with your more recent flag changes.

Reply:
Fixed in [`9b582f9` (#210)](https://github.com/chainguard-dev/bincapz/pull/210/commits/9b582f9874505df5023c7beff8bdb9a24e993778).